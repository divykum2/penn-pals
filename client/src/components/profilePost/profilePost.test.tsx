import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import {BrowserRouter as Router} from 'react-router-dom';
import ProfilePost from './profilePost';

test('render post 1', () => {
    render(<Router><ProfilePost postid={1} /></Router>);

    expect(screen.getByText("Ipsa ducimus soluta architecto.")).toBeInTheDocument();
})

test('Activity feed matches snapshot', () => {
    const component = renderer.create(<Router><ProfilePost/></Router>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});