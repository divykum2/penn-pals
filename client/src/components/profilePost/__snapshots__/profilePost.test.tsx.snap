// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activity feed matches snapshot 1`] = `
<div>
  <p
    className="MuiTypography-root MuiTypography-body1 css-m6yzyh-MuiTypography-root"
  />
  <p
    className="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
  />
</div>
`;
